weasel/_util/errors.py

```diff
@@ -0,0 +1,11 @@
+class ErrorsWithCodes(type):
+    def __getattribute__(self, code):
+        msg = super().__getattribute__(code)
+        if code.startswith("__"):  # python system attributes like __class__
+            return msg
+        else:
+            return "[{code}] {msg}".format(code=code, msg=msg)
+
+
+class Warnings(metaclass=ErrorsWithCodes):
+    W091 = "Could not clean/remove the temp directory at {dir}: {msg}."
```

Comment:
Shouldn't we start again from zero? Any other decision would mean looking at the `weasel` codes any time anyone adds a new error or warning to `spaCy`, which practically nobody is likely to think of doing.

Reply:
I think that originally the plan was to release weasel with 3.5 or another 3.x series release, where keeping codes consistent would be a good idea. Since the plan is now to release it later, it makes sense to renumber from 1, or to drop the numbering convention.